packages/swapper/src/swappers/RelaySwapper/utils/types.ts

```diff
@@ -0,0 +1,237 @@
+  appFees?: AppFee[]
+  gasLimitForDepositSpecifiedTxs?: number
+  userOperationGasOverhead?: number
+  forceSolverExecution?: boolean
+}
+
+export type RelayToken = {
+  chainId: number
+  address: string
+  symbol: string
+  name: string
+  decimals: number
+  metadata: {
+    logoURI: string
+    verified: boolean
+    isNative: boolean
+  }
+}
+
+export type CallFees = {
+  gas: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayer: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayerGas: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  relayerService: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  app: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+}
+
+export type QuoteDetails = {
+  currencyIn: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  currencyOut: {
+    currency: RelayToken
+    amount: string
+    amountFormatted: string
+    amountUsd: string
+    minimumAmount: string
+  }
+  totalImpact: {
+    usd: string
+    percent: string
+  }
+  rate: string
+  slippageTolerance: {
+    origin: {
+      usd: string
+      value: string
+      percent: string
+    }
+    destination: {
+      usd: string
+      value: string
+      percent: string
+    }
+  }
+  timeEstimate: number
+  userBalance: string
+}
+
+// https://github.com/reservoirprotocol/relay-kit/blob/main/packages/sdk/src/types/Execute.ts
+export type Execute = {
+  errors?: {
+    message?: string
+    orderId?: string
+  }[]
+  fees: CallFees
+  details: QuoteDetails
+  error?: any
+  refunded?: boolean
+  steps: {
+    error?: string
+    errorData?: any
+    action: string
+    description: string
+    kind: 'transaction' | 'signature'
+    id: string
+    requestId?: string
+    depositAddress?: string
+    items?: {
+      status: 'complete' | 'incomplete'
+      // @TODO: Add instructions and addressLookupTableAddresses for Solana
+      // and psbt for UTXO
+      data: {
+        from: string
+        to: string
+        data: string
+        value: string
+        chainId: number
+        gas: string
+        maxFeePerGas: string
+        maxPriorityFeePerGas: string
+        depositAddress: string
+      }
+      orderIndexes: number[]
+      orderIds: string[]
+      error?: string
+      txHashes?: {
+        txHash: string
+        chainId: number
+        isBatchTx?: boolean
+      }[]
+      internalTxHashes?: {
+        txHash: string
+        chainId: number
+        isBatchTx?: boolean
+      }[]
+      errorData?: any
```

Comment:
Remove me. This isn't part of the success response type, and this is required for the error response type
<img width="497" alt="image" src="https://github.com/user-attachments/assets/bbdcd072-3c5b-42c8-8d13-7294ec528aa5" />

Reply:
Removed in [9535276](https://github.com/shapeshift/web/pull/9188/commits/95352767db00b2aaec3c108b3cacfcabd09b6e4c)

Also created an issue to handle those errors properly

Reply:
I'm fine with that - Axios will throw with non-200, we can handle accordingly as a follow-up